conf/playReleases.json

```diff
@@ -286,6 +286,12 @@
             "name": "Streamy",
             "version": "2.5.0",
             "date": "Mar 4 2016"
+        },
+		{
```

Comment:
@xael-fry Please fix this opening bracket, thanks!

Reply:
fix it